jekyll/_cci2/oidc-tokens-with-custom-claims.adoc

```diff
@@ -0,0 +1,81 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Customize claims in the OIDC token
```

Comment:
```suggestion
= On-demand OIDC tokens with custom claims
```

Reply:
Is this how we want to refer to the new functionality? If this is correct could you also update the filename to match?

Reply:
We decided to change this to OIDC tokens with custom claims.